src/.vuepress/config.js

```diff
@@ -150,6 +150,10 @@ export default {
             collapsible: true,
             children: [
               {
+	        text: 'Partitioning Instructions for Dualbooting',
```

Comment:
shouldnt this be "dual booting"?

Reply:
Yes, I don't know why I wrote it like that